test_specs/features/step_definitions/API_steps.rb

```diff
@@ -23,21 +22,128 @@
   end
 end
 
+Given(/^I post an idea on "([^"]*)" community with Title: "([^"]*)", Category: "([^"]*)", Tags: "([^"]*)" and "([^"]*)" parameters$/) do |site, idea_title, category_title, tags, params|
+# Get response for the Rest API Call and print it raw
+#   response = @apiutil.postIdea(site, idea_title, category_title, tags, '', params)
+  ideaAPI = IdeaAPI.new(true,'superadmin')
+  response = ideaAPI.postIdea(site, idea_title, category_title, tags, params)
+  puts "\n\nRAW POST CALL RESPONSE: #{response}\n\n"
+
+  # Verify that Code Response expected is 200
+  unless response.code == 201
```

Comment:
This should not be here. There should be a then in the steps that assert for the required error code and stuff.